src/electron/preload.ts

```diff
@@ -137,7 +138,14 @@ const api: Sandbox = {
   },
 
   writeFile: ({ filePath, buffer }) => {
-    fs.writeFileSync(filePath, new DataView(buffer));
+    try {
```

Comment:
なぜthrowではなくreturnなのかの説明があるとわかりやすいかもと感じました！

```suggestion
    // throwだと`.code`の情報が消えるのでreturn
    try {
```